.github/Compiler Guide.md

```diff
@@ -0,0 +1,74 @@
+# Using the Sector File Compiler
+
+## Introduction
+
+The Sector File Compiler is the open-source tool that is used to transform the files that make up the Sector File repository into the .sct, .ese, and .rwy files that Euroscope uses. The link to the repository is [here](https://github.com/VATSIM-UK/sector-file-compiler), and if you should have any issues with the Compiler not working as intended (a verifiable bug), or suggestions to improve it, then do feel free to leave an issue on that repository. If you have questions about its use or aren't quite sure how to do something, do put a message in the #sector_file_development channel in the VATSIM UK Discord.
+
+The Compiler, aside from just merely putting the files into the correct places in the Sector File, is also capable of validating input. This makes it incredibly useful, bordering on essential, for developing 'correct' sector files that work as intended and catching very understandable human error when dealing with Euroscope's esoteric syntax requirements. For example, coordinates entered in the wrong format will produce the following message:
+![image](https://user-images.githubusercontent.com/14115426/208726651-1ff76baa-a197-4120-acb6-1e9f7061d013.png)

```

Comment:
I personally found this image a bit too wide to view properly in the markdown viewer thing I was using, I wonder if might be better to have as a code quote type thing? But upto you which you think is best.

Reply:
Same image appears at bottom too I think, so would apply there too if you wanted to change.

Reply:
Remade it - let me know if it still seems to wide

Reply:
That's better, thanks.